sccm-ps/ConfigurationManager/New-CMTSRule.md

```diff
@@ -0,0 +1,250 @@
+---
```

Comment:
Metadata block should include the following lines starting on line 1:

```
---
title: New-CMTSRule
titleSuffix: Configuration Manager
description: Creates a new rule on a task sequence step in Configuration Manager.
ms.date: 05/07/2019
ms.prod: configuration-manager
ms.technology: configmgr-other
ms.topic: conceptual
author: aczechowski
ms.author: aaroncz
manager: dougeby
---
```

I'm hoping that by setting me as author, our automation bot will let us reopen this PR! 🤞